revolv/payments/models.py

```diff
@@ -9,7 +11,7 @@ class AdminRepaymentManager(models.Manager):
     Manager for AdminRepayment.
     """
 
-    def repayments(self, admin=None, project=None, queryset=None):
+    def repayments(self, admin=None, project=None, queryset=None, start_date=datetime.datetime.min, end_date=datetime.datetime.max):
```

Comment:
May be better to default these to None? That way if they're not provided, the query doesn't care about time restraints at all